### ISSUE TITLE
Show all open tabs in the UI instead of just unstashed tabs

### ISSUE THREAD

**Issue by josh-berry** [josh-berry/tab-stash#116]
### Problem Statement
Sometimes it would be nice to be able to use Tab Stash as a more general-purpose tab manager, for finding, sorting, or closing open tabs that aren't part of any stash.

### Preferred Solution(s)
Provide an option to show all open tabs instead of just "Unstashed Tabs" in the UI, regardless of whether tabs are already stashed or are even stashable.  Give users the ability to manage these tabs through the Tab Stash UI as an alternative to the built-in tab bar.

Note: This should be optional (either opt-in or opt-out, unclear which is better) and not strictly a behavior change.

### Additional Context
This is a placeholder issue to discuss this experimental feature

<hr>
<em>Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
*Known issue:* The Tab Stash tab itself does not appear in the list of open tabs.

This is because the Tab Stash tab cannot itself be stashed, and there is a problem where dragging and dropping an unstashable tab into a stash will cause the UI to show the tab as having been stashed when it wasn't.

**Comment by josh-berry** [comment]
**Known issue:** Pinned tabs are not shown. (This isn't yet implemented.)

I'm not yet sure what the best way to represent them is, or how drag-and-drop of pinned tabs should work. They don't typically participate in stashing, either, so the same caveats for unstashable tabs apply here as well.

**Comment by WebMechanic** [comment]
Hello,
just wanted to give you some feedback on this. 
TL;DR: ♥

I've been using this over the last couple of days and it's **really, really useful** to change the order of all open tabs w/o actually *activating* and *reviving* them as if you drag a tab someplace else the FF way.

If you ctrl+click a link to open in a new tab it adheres to Murphy's law and "always" ends up at the wrong position -- no matter the default setting for this behaviour.

I don't use pinned tabs so I can't report on that.

Anyhow it would be great to have a toogle for this somewhere in the main GUI to switch between "Open Tabs" and "Unstashed Tabs" at will. I dont' think one needs to have both visible at the same time, as the switch happens instantly.
The toggle could sit in the Options Menu

**Comment by josh-berry** [comment]
@WebMechanic Thanks, that’s all good to know! Good point about the toggle; I’ll think about how to do this.

**Comment by tomasz13nocon** [comment]
Great feature!
It would be useful to have some visual indication that an open tab is stashed.

**Comment by josh-berry** [comment]
> Great feature! It would be useful to have some visual indication that an open tab is stashed.

Indeed! Amusingly enough, I was already working on this. 😆

**Comment by josh-berry** [comment]
This should be in good shape for the next release!

Note that after you upgrade, whatever previous setting you had will be forgotten--you'll just have to turn it on again once and then it will be remembered going forward. I decided to change how the option was being stored as part of making the feature not-experimental (and to allow for more variants of the setting in the future).

**Comment by mjbamford** [comment]
Just discovered this new functionality in the latest release... fantastic! Love how there is a visual indication when a tab has been stashed in an existing group, and the hover over the new icon to see what stash that is.
